build.sbt

```diff
@@ -4,8 +4,8 @@ name := "npm"
 scalaVersion := "2.10.6"
 
 libraryDependencies ++= Seq(
-  "com.typesafe" %% "jse" % "1.2.0",
-  "org.webjars" % "npm" % "2.14.14",
+  "com.typesafe" %% "jse" % "1.2.1",
+  "org.webjars" % "npm" % "4.2.0",
```

Comment:
Hmm. Unfortunately it doesn't seem as simple as just upgrading the version number. I tried and the tests failed. I'll leave that for another PR as it could be complicated by itself